### PR TITLE
Tips to help with advertiser onboarding

### PR DESCRIPTION
**Pull request by davidfischer**
- A getting started for new advertisers
- Otherwise, a screen for returning but not current advertisers
- Lastly, statistics for advertisers currently running ads

### CODE REVIEW

File: adserver/templates/adserver/advertiser/overview.html [hunk 1]
```diff
@@ -62,11 +62,51 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
...
+        <div class="col">
+          <h3>{% trans 'There are a few steps to getting started with your first ad campaign with us.' %}</h3>
+          <ol class="mt-5 lead">
+            <li>
+              {% url "flight_list" advertiser.slug as flight_list_url %}
+              <p>{% blocktrans %}View the upcoming <a href="{{ flight_list_url }}">flights</a> set up for you and make sure the content and geographic targeting matches your customer focus. The price of your campaign depends on your chosen targeting.{% endblocktrans %}</p>
+            </li>
+            <li>
+              {% url "support" as support_url %}
+              <p>{% blocktrans %}Once your flights are set up, we will issue you an invoice. You can upload your ad creatives to your flights.{% endblocktrans %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/advertiser/overview.html:77]: What is `set up`? Do they need to take an action here? Email us?

File: adserver/templates/adserver/advertiser/overview.html [hunk 2]
```diff
@@ -62,11 +62,51 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
...
+            </li>
+            <li>
+              <p>{% blocktrans %}Once the invoice is paid, we can start your flight and start sending traffic to you. We can also set your flight to start automatically on a specific date.{% endblocktrans %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/advertiser/overview.html:80]: This is very passive voice with too many `can`'s. It feels a bit like we're trying to explain the process, but also explain additional features? 

I think this copy can be improved to be clearer on what *they* need to do, and probably have less somewhat random feature references? It seems like that should all be handled somewhere else, and we could link to it.

File: adserver/templates/adserver/advertiser/overview.html [hunk 3]
```diff
@@ -62,11 +62,51 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
...
+          <h3>{% trans 'Welcome back.' %}</h3>
+          <ul class="mt-5 lead">
+            <li>
+              {% url "advertiser_report" advertiser.slug as report_url %}
+              {% url "flight_list" advertiser.slug as flight_list_url %}
+              <p>{% blocktrans %}You can always look at <a href="{{ flight_list_url }}">past flights</a> or check your past <a href="{{ advertiser_report }}">ad performance</a> for any timeframe.{% endblocktrans %}</p>
+            </li>
+            <li>
+              {% url "support" as support_url %}
+              <p>{% blocktrans %}If you're interested in running a new ad flight, please <a href="{{ support_url }}?subject=New+campaign&body=I+am+interested+in+running+a+new+campaign.">get in touch</a>.{% endblocktrans %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/advertiser/overview.html:98]: 👍

**Review by ericholscher — COMMENTED** [review]
This is a good upgrade, but the copy for new people needs some work. It would be great if we actually had a checklist with items checked off (eg. ✔️  your flights are configured, ✔️  your invoice has been paid). But even just starting with clarifying the language around what is their responsibility, and what actions they need to take would be great.

File: adserver/templates/adserver/advertiser/overview.html (hunk 1 above)
Comment (davidfischer) [reply adserver/templates/adserver/advertiser/overview.html:77]: Generally not. I'll clarify.

File: adserver/templates/adserver/advertiser/overview.html (hunk 2 above)
Comment (davidfischer) [reply adserver/templates/adserver/advertiser/overview.html]: I reworded it:
Screenshot from 2022-05-03 14-14-43

File: adserver/templates/adserver/advertiser/overview.html [hunk 4]
```diff
@@ -67,17 +67,28 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
...
               <p>{% blocktrans %}View the upcoming <a href="{{ flight_list_url }}">flights</a> set up for you and make sure the content and geographic targeting matches your customer focus. The price of your campaign depends on your chosen targeting.{% endblocktrans %}</p>
             </li>
             <li>
-              {% url "support" as support_url %}
-              <p>{% blocktrans %}Once your flights are set up, we will issue you an invoice. You can upload your ad creatives to your flights.{% endblocktrans %}</p>
+              {% if are_ads_set_up %}
+                <span class="fa-li fa fa-check"></span>
+              {% else %}
+                <span class="fa-li fa fa-square-o"></span>
+              {% endif %}
```
Comment (ericholscher) [inline adserver/templates/adserver/advertiser/overview.html:81]: Love this 💯

File: adserver/templates/adserver/advertiser/overview.html [hunk 5]
```diff
@@ -67,17 +67,28 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
...
             </li>
             <li>
-              <p>{% blocktrans %}Once the invoice is paid, we can start your flight and start sending traffic to you. We can also set your flight to start automatically on a specific date.{% endblocktrans %}</p>
+              {% if has_paid_invoice %}
+                <span class="fa-li fa fa-check"></span>
+              {% else %}
+                <span class="fa-li fa fa-square-o"></span>
+              {% endif %}
+              {% url "advertiser_stripe_portal" advertiser.slug as billing_portal_url %}
+              <p>{% blocktrans %}Pay your <a href="{{ billing_portal_url }}">first invoice</a> and we will start sending traffic to you.{% endblocktrans %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/advertiser/overview.html:91]: Will they always have an invoice if they have flights setup? They don't need to email us again after we create their flights & invoice together?

**Review by ericholscher — APPROVED** [review]
Love this checklist 👍

File: adserver/templates/adserver/advertiser/overview.html (hunk 5 above)
Comment (davidfischer) [reply adserver/templates/adserver/advertiser/overview.html:91]: Usually not but sometimes. Sometimes we do create their account before we've finalized the targeting especially if they aren't sure yet. The first flight is created during the account creation process.

**Pull request merged**